django_nyt/utils.py

```diff
@@ -84,9 +86,9 @@ def notify(
     if app_settings.NYT_ENABLE_CHANNELS:
         from django_nyt import subscribers
 
-        subscribers.notify_subscribers(objects, key)
+        subscribers.notify_subscribers(notifications, key)
 
-    return len(objects)
+    return notifications
```

Comment:
I think this is fine for the moment. Of course, an implementation that creates 10,000 notifications should do this differently, and probably async. So that needs a different approach all-together.